app/src/main/java/com/yml/womensafety/navigationdrawer/home/HomePageActivity.kt

```diff
@@ -50,14 +51,23 @@ class HomePageActivity : AppCompatActivity() {
         })
 
         cvEscapeThreat.setOnClickListener {
-            tipsToEscapeFragment = TipsToEscapeFragment()
+//            tipsToEscapeFragment = TipsToEscapeFragment()
+//            supportFragmentManager
```

Comment:
Please remove commented code

Reply:
Done

Reply:
I dont see the changes reflecting here. Are the changes pushed yet @Annapoornakamath98 ?